Ladder.java

```diff
@@ -9,25 +10,29 @@ public Ladder(int numberOfPeople, int ladderDepth) {
         this.ladder = new String[ladderDepth][1];
         this.random = new Random();
 
-        for (int i = 0; i < ladderDepth; i++) {
-            ladder[i][0] = initLadderLow(numberOfPeople + (numberOfPeople - 1));
-        }
+        IntStream.range(0, ladderDepth)
+                .forEach(row -> ladder[row][0] = initLadderRow(numberOfPeople * 2 - 1));
     }
 
-    private String initLadderLow(int LadderLine) {
+    private String initLadderRow(int ladderLine) {
         StringBuilder sb = new StringBuilder();
 
         // 열이 짝수라면 "|", 홀수라면 " " or "-" 입력
-        for (int i = 0; i < LadderLine; i++) {
-            if (i % 2 == 0) {
-                sb.append("|");
-                continue;
-            }
-            sb.append(initRandomLine());
-        }
+        IntStream.range(0, ladderLine).forEach(line -> determineLineShape(line, sb));
         return sb.toString();
     }
 
+    private StringBuilder determineLineShape(int line, StringBuilder sb) {
+        if (validateEven(line)) {
+            return sb.append("|");
+        }
+        return sb.append(initRandomLine());
+    }
+
+    private boolean validateEven(int number) {
```

Comment:
```suggestion
    private boolean isEven(int number) {
```

validate는 짝수가 아니면 안 된다라는 느낌이 있어서 그냥 is가 나은 느낌입니다.